TemplePlus/tig/tig_startup.cpp

```diff
@@ -287,6 +287,7 @@ void TigInitializer::LoadDataFiles() {
 		else {
 			moduleBase = moduleName;
 		}
+		modSupport.DetectZMOD();
```

Comment:
This bothered me too, will get back to you

Reply:
You'll throw an exception if you call IsZmod before ModSupport is intialized, so you'll know about it at least. Ok.